package.json

```diff
@@ -54,7 +54,11 @@
     "node-sass": "3.3.3",
     "pako": "0.2.8",
     "po2icu": "git://github.com/LLK/po2icu.git#develop",
+    "react": "0.14.0",
```

Comment:
`react`, `react-dom`, and `react-intl` can be removed from the package as they are included as `externals` via Webpack.